src/utils/object.js

```diff
@@ -24,6 +24,9 @@ export const evaluate = (template, model) => {
 };
 
 export const findIndexById = (array, id) => {
-  if (!array || !array.length || (!id && id !== '0' && id !== 0)) return -1;
-  else return array.findIndex(item => item.id === id);
+  if (!array || !array.length || (!id && id !== '0' && id !== 0)) {
+    return -1;
+  } else {
```

Comment:
No need in `else`

Reply:
left only a condition to prevent an error:
`export const findIndexById = (array, id) => {
  if (!Array.isArray(array)) {
    return -1;
  } else {
    return array.findIndex(item => item.id === id);
  }
};`

Reply:
done

Reply:
done